srcpkgs/mkinitcpio-encryptssh/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'mkinitcpio-encryptssh'
 pkgname=mkinitcpio-encryptssh
 version=0.16
```

Comment:
https://github.com/grazzolini/dropbear_initrd_encrypt

This hook was split into 3 different ones, is there any reason to keep it @thypon  ?

Reply:
Any change needed?